src/IO/InputReader.php

```diff
@@ -138,8 +138,8 @@ public function ask(string $question, $yesValue = true, $noValue = false, bool $
     }
 
     /**
-     * @param list<string|int> $answers
-     * @param list<string>     $choices
+     * @param string[]     $answers
```

Comment:
According to https://github.com/composer/composer/blob/2.4.4/src/Composer/IO/ConsoleIO.php#L320, `$answers` is always a list of strings. Was there any reason why you switched from `list` to `array`? IMHO, it should be `list<string>` (without `int`).

Reply:
I most certainly agree. It can only be an `array<int, string>`. However, according to the static code analysis the param `$answer` is of `array<string>`.

```
------ -----------------------------------------------------------------------------------------------------------------------
  Line   src/IO/InputReader.php
 ------ -----------------------------------------------------------------------------------------------------------------------
  114    Parameter #1 $answers of method CPSIT\ProjectBuilder\IO\InputReader::parseMultipleAnswers() expects list<int|string>,
         array<string> given.
 ------ -----------------------------------------------------------------------------------------------------------------------
```

Reply:
Okay, it seems like the return annotation in Composer's `IOInterface` is not explicit enough in this case. I'll prepare a PR to suggest the `list<string>` as return type instead of `string[]`. For the time being, we can leave the array syntax instead.

Reply:
See composer/composer#11171